internal/compiler-bridge/src/main/scala_2.13/xsbt/ConsoleInterface1.scala

```diff
@@ -16,24 +16,24 @@ import scala.tools.nsc.interpreter.IMain
 import scala.tools.nsc.interpreter.shell.{ ILoop, ShellConfig, ReplReporterImpl }
 import scala.tools.nsc.{ GenericRunnerCommand, Settings }
 
-class ConsoleInterface {
-  def commandArguments(
+class ConsoleInterface1 extends xsbti.compile.ConsoleInterface1 {
+  override def commandArguments(
       args: Array[String],
       bootClasspathString: String,
       classpathString: String,
       log: Logger
   ): Array[String] =
     MakeSettings.sync(args, bootClasspathString, classpathString, log).recreateArgs.toArray[String]
 
-  def run(
+  override def run(
       args: Array[String],
       bootClasspathString: String,
       classpathString: String,
       initialCommands: String,
       cleanupCommands: String,
       loader: ClassLoader,
       bindNames: Array[String],
-      bindValues: Array[Any],
+      bindValues: Array[AnyRef],
```

Comment:
What drives this change?

Reply:
Because it now extends Java interface `xsbti.compile.ConsoleInterface1` with `Object` array. `Object` is `AnyRef` so without this change it will not compile. Note that call-site has been casting to `val values = values0.toArray[Any].asInstanceOf[Array[AnyRef]]`.